### DISCUSSION TITLE
Hybrid meetings: best way to show in listings?

### DISCUSSION THREAD

**Discussion by OSAwebsite** [code4recovery/12-step-meeting-list#525] [Q&A] [answered]
Hi team,
I am starting to get requests by groups who are contemplating having hybrid meetings.
Everyone has a different interpretation of the possible models and I am wondering if there are things in place to handle them.

The word hybrid can mean many things.
1: A meeting is held in person and at the same time that exact same meeting can be attended via Zoom
2: A meeting is held in person without online access, but there is another meeting, possibly on another day and at a different time, that can be attended online
3: There is an in person meeting and, at the exact same day and time, there is a concurrent online meeting. Both meeting have different people attending

I was thinking making separate listings for the meetings that have their online part on a different day and/or time than the in person. Because, those would simply be online meetings by the same group.

The only real hybrid meeting would be model 1, where people can attend an in-person meeting online. For that, it would be helpful to have a checkbox option with the word 'hybrid'. Is that under development at all?

Any suggestions and ideas about handling this differently or in a better way are very welcome.

Thanks,
Nicky

**Comment by tech2serve** [comment, marked as answer]
Hi Nicky,

Good news! TSML 3.11 already supports hybrid meetings, using definition 1. Here is how we define a hybrid meeting (although, I'm going to back off of that term a little bit in a moment): If an in-person meeting and a zoom/telephone meeting share the same chairperson (at the same time) and the people in both venues can share back and forth, the meeting is a hybrid meeting.

For meetings that are held at the same time, but for which they have different chairs and attendees at each cannot see or hear attendees at the other, those are different meetings and should be listed twice.

TSML doesn't emphasize the term hybrid but it does use it in the notes. To create a hybrid meeting, answer "yes" to the question about whether or not the meeting can be attended in person. Then also complete the online details for that meeting, and TSML will treat it as hybrid. It is that easy.

Hopefully this answers your question.

**Reply by tech2serve** [reply]
Let me clarify (as I wasn't clear). By notes, I'm referring to the instructions that show up here:

<img width="679" alt="image" src="https://user-images.githubusercontent.com/31904316/133519940-a0cd8346-04ec-432e-8420-8a29927fe89a.png">

There is not a simple way to search just for hybrid meetings because we don't believe this is a common user search. However, you can add `&tsml-attendance_option=hybrid` to the URL and it should list only hybrid meetings. This is effective for admins to create menu links for meetings with a particular attendance option. Tbh, this feature may not be supported in the future.

**Comment by OSAwebsite** [comment]
That sounds logical. The only thing I am not sure I am understanding is 'we use the term in the notes field'. You mean it is put in there automatically even if I don't put it there? 

Thanks for the other clarification. Glad we are on the same page!

Thanks for all you hard work!!!

**Comment by OSAwebsite** [comment]
As a by the way: I can perform a search using the word hybrid to find the meetings where I have used that term in the notes.
However, it does not seem to work to look for the tag 'in-person and online'. Is there a way to make that happen?

**Reply by tech2serve** [reply]
Let me start by apologizing that I missed and did not respond to your additional questions on this thread. No excuse on my part.

We have the word “hybrid” in the instructions for entering a meeting, but we don’t automatically include it in any user-side notes. 

The main reason for not doing this is because some people like the word hybrid while others don’t. We encourage webmasters such as yourself to include the word “hybrid” or “combined” or whatever fits in your area, in the meeting notes which will then be searchable.

**Comment by OSAwebsite** [comment]
Ok. Thanks for letting me know. I will try this.

On Wed, Sep 15, 2021 at 6:52 PM Tech2Serve ***@***.***> wrote:

> Let me clarify (as I wasn't clear). By notes, I'm referring to the
> instructions that show up here:
>
> [image: image]
> <https://user-images.githubusercontent.com/31904316/133519940-a0cd8346-04ec-432e-8420-8a29927fe89a.png>
>
> There is not a simple way to search just for hybrid meetings because we
> don't believe this is a common user search. However, you can add
> &tsml-attendance_option=hybrid to the URL and it should list only hybrid
> meetings. This is effective for admins to create menu links for meetings
> with a particular attendance option. Tbh, this feature may not be supported
> in the future.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/code4recovery/12-step-meeting-list/discussions/525#discussioncomment-1335815>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AOIWBF4DHI3OIA4C2F4KBN3UCEPSJANCNFSM5D6IUMOA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>
-- 


-
Yours in service,
Nicky D
Ontario South Website Coordinator
al-anon.alateen.on.ca

*This too will pass...*

**Comment by OSAwebsite** [comment]
Thank you. I have done just that.


-
Yours in service,
Nicky D
Ontario South Website Coordinator
al-anon.alateen.on.ca

*This too will pass...*
